gulp/typescript.js

```diff
@@ -53,4 +55,17 @@ module.exports = (gulp, plugins, blueprint) => {
             tsResult.dts,
         ]).pipe(blueprint.dest(project));
     });
+
+    const bundleTaskNames = blueprint.projectsWithBlock("typescript").map((project) => {
+        const taskName = `typescript-bundle-${project.id}`;
```

Comment:
soooo i had a similar thought, but it _does_ have to do with the `typescript` config block, which is the naming convention for compile tasks: `block-name-project`. i can't use `blueprint.task()` directly here to enforce that pattern because that method doesn't support callback task definitions, so i had to spell it out by hand.

that said, i don't feel _too_ strongly.